BookPlayer/Player/Player Screen/PlayerViewModel.swift

```diff
@@ -430,7 +435,15 @@ class PlayerViewModel: ViewModelProtocol {
         }
       )
     )
-
+    
+    actions.append(
+      BPActionItem(
+        title: "sleeptimer_option_sticky".localized,
+        handler: {
+          SleepTimer.shared.setTimer(.sticky)
```

Comment:
This would instead call the setter for the sticky preference mentioned in another comment